### ISSUE TITLE
[EPUB 3.3] [suggestion] Note on accessibility and orientation for FXL

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/epub-specs#1736]
Perhaps it might make sense to include a note saying that blocking the orientation may create problems for content accessibility?

https://www.w3.org/TR/epub-33/#orientation

**Comment by mattgarrish** [comment]
I expect this will be covered in the forthcoming FXL accessibility practices document.

**Comment by gregoriopellegrino** [comment]
Fine, is it possible to tag in a way that we don't forget about this? :)

**Comment by mattgarrish** [comment]
Ya, I wouldn't close this issue until it's definitely addressed. I've labeled it for the FXL accessibility document as it's not something the main group needs to take up.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2022-02-17-epub#section3) on 2022-02-17

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

### 3. FXL Limitations.

_See github issue [epub-specs#1733](https://github.com/w3c/epub-specs/issues/1733)._




_See github issue [epub-specs#1736](https://github.com/w3c/epub-specs/issues/1736)._




**Wendy Reid:** not a lot of discussion on these, but they are still open, so we want to mention.  
… gpellegrino did review of a11y documentation last year.  
… he raised 2 issues specifically about FXL a11y.  
… one is about there being limitations on FXL a11y right now, not able to make fully accessible FXL document right now.  
… other one is about accessibility and orientation, there will be a note in best practices that blocking orientation can impede readability.  
… if no comments, both gpellgrino's comments will be reflected in note.  

**Dave Cramer:** my understanding is that the problem which cannot be avoided in making FXL accessible content is that font size is fixed.  
… is there accessibility metadata around there?.  

**Wendy Reid:** i think so, but I can't recall specific accessibility hazard text at the moment.  
… but yes, inability to modify text size, font face.  
… we will be mapping some of the statements in the best practice note to WCAG - one of these is about scrolling, i.e. having to scroll both horizontally and vertically to read content if zoomed in.  

**Dave Cramer:** i could definitely make an FXL that doesn't specify fonts, but not sure if RS would allow user to choose a font.  

**Brady Duga:** no, the whole point of FXL is that you're not supposed to change that.  

**Dave Cramer:** does the CSS positioning spec say that you can use these things to make inaccessible documents?.  

**Wendy Reid:** there is an accessibility feature call display transformability, but not corresponding accessibility hazard for no display transformability.  
… but I will ask authors of a11y spec.  

**Dave Cramer:** historically epub has allowed a lot of different ways to achieve a particular goal, and we've never taken the position that every epub must be accessible to every user.  
… for example, standard alone audio books are very useful for accessibility, but also useless to a specific segment of people.  
… this seems like we're taking a step towards saying FXL is an invalid way of presenting content to users.  

**Wendy Reid:** this is all non-normative, but i'm phrasing the note as "the limits of FXL a11y".  
… it is possible to do a lot of a11y work for a lot of people, but there are limitations that make it so that some segments will not be served.  

**Shinya Takami (高見真也):** right now we do not have requirement to make epub accessible, so it will be hard to realize accessible epubs i think.  

**Wendy Reid:** isn't a version of epub a11y 1.1 adopted as JIS standard?.  

**Murata Makoto:** it will be later this year, in aug/sep.  

**Dave Cramer:** okay, what wendyreid has proposed sounds good to me.  

</details>
